Sources/schema-gen/PropertyDecl.swift

```diff
@@ -56,29 +56,31 @@ struct PropertyDecl {
     }
 
     var documentation: String?
-    if let leadingTrivia = variable.leadingTrivia, leadingTrivia.count >= 3 {
-      var lines: [String] = []
-      for trivia in leadingTrivia {
-        let triviaString = String(describing: trivia)
-          .trimmingCharacters(in: .whitespaces)
-
-        if triviaString.hasPrefix("/// ") {
-          lines.append(String(triviaString.dropFirst(4)))
-        } else if triviaString == "///" {
-          lines.append("")
-        }
-      }
+    let leadingTrivia = variable.leadingTrivia
+    guard leadingTrivia.count >= 3 else {
+      return .failure(.notIdentifierBinding)
+    }
+
+    var lines: [String] = []
+    for trivia in leadingTrivia {
+      let triviaString = String(describing: trivia)
+        .trimmingCharacters(in: .whitespaces)
 
-      if !lines.isEmpty {
-        documentation = lines.joined(separator: "\n")
+      if triviaString.hasPrefix("/// ") {
+        lines.append(String(triviaString.dropFirst(4)))
+      } else if triviaString == "///" {
+        lines.append("")
       }
     }
 
+    if !lines.isEmpty {
+      documentation = lines.joined(separator: "\n")
+    }
+
     var modifiers: [String] = []
-    if let modifierList = variable.modifiers {
-      for modifier in modifierList {
-        modifiers.append(modifier.name.withoutTrivia().text)
```

Comment:
It looks like this is the only other usage of `.withoutTrivia()`, in this case since it's not using `description`, this seems like it makes the most sense:
```diff
for modifier in modifierList {
- modifiers.append(modifier.name.withoutTrivia().text)
+ modifiers.append(modifier.name.trimmed.text)
}
```

Reply:
Actually, the `trimmed` usage shouldn't be necessary because the doc for `.text` says: **The text of the token as written in the source code, without any trivia.**